.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,10 +17,12 @@ Closes # <-- _insert Issue number if one exists_
 
 ## Checklist
 
+- [ ] formatted title correctly? (_see [here](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CONTRIBUTING.md#submit-a-pull-request)
```

Comment:
we could also reference the styleguide directly, i.e. https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/styleguide.md